src/Algebra/Module/Properties.agda

```diff
@@ -0,0 +1,47 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of modules.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra                using (Semiring)
+open import Algebra.Module.Bundles using (LeftSemimodule)
+open import Level                  using (Level)
+
+module Algebra.Module.Properties
+  {r ℓr m ℓm : Level} {semiring : Semiring r ℓr}
+  (lsmod : LeftSemimodule semiring m ℓm)
+  where
+
+open import Relation.Nullary          using (¬_)
+open import Relation.Nullary.Negation using (contraposition)
+
+open LeftSemimodule lsmod
+  using ( *ₗ-zeroˡ; *ₗ-zeroʳ; ≈ᴹ-setoid; *ₗ-congˡ; *ₗ-congʳ
+        ; _*ₗ_; 0ᴹ; _≈ᴹ_; _≉ᴹ_)
+  renaming (Carrierᴹ to M)
```

Comment:
No longer need this renaming and the renaming below. So we can probably just `open LeftSemimodule` without the `using`?

Reply:
Yep, thanks!
Fixed and arriving in next push.